apps/website/docs/infrastructure/azure/apim/best-practices.md

```diff
@@ -0,0 +1,119 @@
+---
+sidebar_label: Best Practices
```

Comment:
suggestion: I'd rather have a page titled something like 'Debug APIM Policies' with info about using revisions and VScode. Then another page about subscriptions and products titled 'Manage APIs access and policies'